base/src/test/proto/spine/test/validate/set_once_test.proto

```diff
@@ -34,18 +34,15 @@ option java_package = "io.spine.test.validate";
 option java_outer_classname = "SetOnceTestProto";
 option java_multiple_files = true;
 
-import "spine/people/person_name.proto";
-import "spine/net/url.proto";
-
 message Passport {
     option (entity).kind = ENTITY;
 
     string id = 1; // implicitly: (set_once) = false
 
-    people.PersonName name = 2 [(set_once) = false];
+    string name = 2 [(set_once) = false];
```

Comment:
Please add local value objects for the person's name and photo URL. At least, just with `string value = 1;`, so that the readers who may find these tests see the right way of doing models.